ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -91,6 +91,12 @@ objects:
         value: /tmp/shared/home
       - name: TEST_IN_PRODUCTION
         value: ${TEST_IN_PRODUCTION}
+      - name: RESOURCEGROUP
+        value: e2e-${RESOURCEGROUP_SUFFIX}
+      - name: TEST_IMAGE
+        value: ${TEST_IMAGE}
+      - name: TEST_COMMAND
```

Comment:
Not sure you need this. TEST_COMMAND should already work as is, no?

Reply:
I thought it might be good to be explicit.  I can remove it if you'd prefer.

Reply:
will remove.